packages/zosfiles/src/methods/list/List.ts

```diff
@@ -76,7 +77,18 @@ export class List {
             };
         } catch (error) {
             this.log.error(error);
-            throw error;
+
+            // Throw detailed error message with REST debugging info if that option is included
+            if (options.debugResponse) {
+                throw error;
+            } else {
+                // Isolate the main error message to display
+                const splitMessage: string[] = error.message.split("details:");
+
+                // Throw short error message
+                process.stdout.write("\n--> Use option `--debug-response` or `--dr` with the command to see REST API level debugging info.\n\n");
+                throw Error(splitMessage[splitMessage.length - 1].replace(/\s+$/, ""));
```

Comment:
I believe that we want to avoid writing directly to process.stdout (or stderr) directly.
We should always aim to use the `params.response.console` functions to do so.
However, there is no such concept as a CommandResponse object here since this is an SDK method.

Given that we are throwing an error, maybe we could add the short error message in the `ImperativeError.additionalDetails` property (or some other `ImperativeError.shortDescription` optional property) and let the receiving handler react to such error and print the short message if available.

kudos to @t1m0thyj for helping come up with idea 😋